crates/walrus-service/src/common/config.rs

```diff
@@ -71,19 +71,19 @@ impl SuiConfig {
     }
 
     /// Creates a [`SuiContractClient`] based on the configuration.
+    /// Always use the RPC endpoints supplied in the configuration instead of the wallet's RPC URL
```

Comment:
It seems wrong to not include the wallet's rpc url, no?

Reply:
I thought that url from wallet is deprecated after #2052? It is marked as `deprecated` with notes that `Avoid calling this method as it relies on the Wallet configuration for rpc urls. It's better to pass rpc urls directly to the location where they are needed, from Walrus configuration.`

I can add it but want to make sure what's the plan for the url in wallet.

Reply:
That's fair. If you are confident that the urls we're getting from the alternate source are better, then let's avoid calling `get_rpc_url`.